app/assets/javascripts/lib/util/analytics.js

```diff
@@ -0,0 +1,31 @@
+var Analytics = (function () {
+    'use strict';
+
+    function track(obj) {
+        if (window.ga) {
+            window.ga('send', obj.event, obj.type, obj.data);
+        }
+    }
+
+    Analytics.prototype = {
```

Comment:
Nitpicking: the noun/verb order of event names in `Analytics` isn't consistent:

* **chart**Added
* change**Input**